charts/trino/values.yaml

```
@@ -47,6 +47,57 @@ server:
     maxReplicas: 5
     targetCPUUtilizationPercentage: 50
 
+accessControl: {}
+  # type: configmap
+  # refreshPeriod: 60s
+  # # Rules file is mounted to /etc/trino/access-control
+  # configFile: "/access-control/rules.json"
```

```suggestion
  # configFile: "rules.json"
```

since it already gets mounted to `/etc/trino/access-control` and the path `security.config-file={{ .Values.server.config.path }}/access-control/{{ .Values.accessControl.configFile | default "rules.json" }}` already includes that segment.